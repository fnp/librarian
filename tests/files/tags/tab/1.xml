<strofa>
  A<tab/>B/
  A<tab szer="1" />1/
  A<tab szer="2" />2/
  A<tab szer="3" />3/
  A<tab szer="4" />4/
  A<tab szer="5" />5/
  A<tab szer="6" />6/
  A<tab szer="1em" />1em/
  A<tab szer="2em" />2em/
  A<tab szer="3em" />3em/
  A<tab szer="4em" />4em/
  A<tab szer="5em" />5em/
  A<tab szer="6em" />6em/
  A<tab szer="*" />*/
  Rozszerzony tabulator <wyroznienie>wewnątrz<tab szer="*" />wyróżnienia</wyroznienie>
</strofa>
