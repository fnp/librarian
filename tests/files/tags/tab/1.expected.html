<div id="book-text"><div class="stanza" id="i1">
<a href="#f1" class="wl-num">1</a><div class="wl verse" id="f1">A<span display="inline-block" width="1em"></span>B</div>
<div class="wl verse" id="f2">A<span display="inline-block" width="1em"></span>1</div>
<div class="wl verse" id="f3">A<span display="inline-block" width="2em"></span>2</div>
<div class="wl verse" id="f4">A<span display="inline-block" width="3em"></span>3</div>
<a href="#f5" class="wl-num">5</a><div class="wl verse" id="f5">A<span display="inline-block" width="4em"></span>4</div>
<div class="wl verse" id="f6">A<span display="inline-block" width="5em"></span>5</div>
<div class="wl verse" id="f7">A<span display="inline-block" width="6em"></span>6</div>
<div class="wl verse" id="f8">A<span display="inline-block" width="1em"></span>1em</div>
<div class="wl verse" id="f9">A<span display="inline-block" width="2em"></span>2em</div>
<a href="#f10" class="wl-num">10</a><div class="wl verse" id="f10">A<span display="inline-block" width="3em"></span>3em</div>
<div class="wl verse" id="f11">A<span display="inline-block" width="4em"></span>4em</div>
<div class="wl verse" id="f12">A<span display="inline-block" width="5em"></span>5em</div>
<div class="wl verse" id="f13">A<span display="inline-block" width="6em"></span>6em</div>
<div class="wl verse verse-stretched" id="f14">
<span>A</span><span class="verse-stretched-space"></span><span>*</span>
</div>
<a href="#f15" class="wl-num">15</a><div class="wl verse verse-stretched" id="f15">
<span>Rozszerzony tabulator <em class="author-emphasis">wewnątrz</em></span><span class="verse-stretched-space"></span><span><em class="author-emphasis">wyróżnienia</em></span>
</div>
</div></div>
